components/Form/store.tsx

```diff
@@ -295,7 +296,7 @@ class Store<
   };
 
   public getFieldValue = (field: FieldKey): FieldValue => {
-    return get(this.store, field);
+    return get(this.getFields(), field);
```

Comment:
cloneDeep(get(this.store, field)) ？

Reply:
Yes, your solution has better performance, I had change it.